microsoft/testsuites/network/sriov.py

```diff
@@ -67,6 +69,27 @@ def before_case(self, log: Logger, **kwargs: Any) -> None:
                 enable=True, wait=True, reset_connections=True
             )
 
+    @TestCaseMetadata(
+        description="""
+        This case verifies all services state with Sriov enabled.
+
+        Steps,
+        1. Get overrall state from `systemctl status`, if no systemctl command,
+           skip the testing
+        2. The expected state should be `running`
+        """,
+        priority=1,
+        requirement=simple_requirement(
+            network_interface=features.Sriov(),
+        ),
+    )
+    def verify_services_state(self, environment: Environment) -> None:
+        try:
+            for node in environment.nodes.list():
+                assert_that(node.tools[Systemctl].state()).is_equal_to("running")
+        except UnsupportedDistroException as identifier:
+            raise SkippedException(identifier)
```

Comment:
I'm not sure where `identifier` came from, it's a little confusing. The typical naming convention is `e` or sometimes `err`. You should also use exception chaining in cases like this.

```suggestion
        except UnsupportedDistroException as e:
            raise SkippedException(e) from e
```

Reply:
thanks, updated